### ISSUE TITLE
Bug: Custom FOV Can’t prevent certain FOV changes

### ISSUE THREAD

**Issue by SirAron111** [TheIllusiveC4/CustomFoV#1]
#### Versions (Be specific, do not write "latest"):
* Custom FoV:customfov-1.12.2-1.0
* Forge:14.23.5.2838 
#### Steps to Reproduce:
1.Installe these mods test it out
2.watch it bob around
3.try the Draconic bow on gattling mode and get dizzy 
...
Crash Log: Please use a paste site like [gist](https://gist.github.com/).

**Comment by TheIllusiveC4** [comment]
Ah right, yeah, I forgot to think about other mods affecting the FoV. This should be an easy fix.

**Comment by SirAron111** [comment]
The same problem happens with things like speed boost from tinkers armor Redstone modifier or other mods potions Ender Io bows etc etc the mod NoFOV fixes it nicely but it to cant fix the Draconic Bow and it can’t be adjusted in FOV like yours (turning yours to values of 0 makes it look really funny inverted when going underwater or so btw)

**Comment by TheIllusiveC4** [comment]
> (turning yours to values of 0 makes it look really funny inverted when going underwater or so btw)

Could you elaborate on this?

Edit: Nevermind, I think I see what you mean.

**Comment by TheIllusiveC4** [comment]
> things like speed boost from tinkers armor Redstone modifier

I've just tested this myself and the fov changes work fine, as it should since Custom FoV was originally designed for that specific use.

**Comment by TheIllusiveC4** [comment]
So the initial problem is actually more of a feature request than a bug, and I forgot about this initially, because the static fov config option specifically states that it will only affect vanilla changes. This precludes modded items like the Ender Bow/Draconic Bow and such. However, I've added in a new config option called "Super Static FoV" that will override everything, modded and vanilla alike. This should satisfy your use-case.

I've also addressed the issue with the underwater fov changes acting strangely with certain values.

I could not find anything wrong with the speed fov changes, modded or otherwise.
